test/tests/pbs_smoketest.py

```diff
@@ -1361,7 +1361,7 @@ def delete_resource_helper(self, r, t, f, c, obj_type, obj_id):
                 if d and len(d) > 0:
                     self.assertFalse(ar in d[0])
 
-    @timeout(720)
+    @timeout(1200)
```

Comment:
This might be out of scope for this PR, but if this test takes so long (1200s is >11m), is it a good candidate for a smoke test?  Should we move it to a functional test?

Reply:
@bhroam actually test_resource_* in SomkeTest are qmgr resource interface regression, it should be part of interfaces test (We already have a ticket for it in the old issue tracker: https://pbspro.atlassian.net/browse/PP-1226)

Reply:
Hi @bhroam ,@hirenvadalia i will remove test_resource_create and test_resource_delete tests from SmokeTest test suite and add slimmed test for resource. 
Please let me know we have TestQmgr test suite in functional. Is that fine to move these tests to TestQmgr or i have to create separate test suite in interfaces as mentioned by @hirenvadalia .

Reply:
I am ok moving into TestQmgr. Also if possible I would suggest dividing test_resource_create and test_resource_delete into multiple that way will have multiple short-running tests instead of long-running tests.

Reply:
I'm fine with this.  Thanks for doing this @dinesh1306